SW.SecurityService.Core/Models/AuthenticationAnswer.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace SW.SecurityService.Core.Models
+{
+    public class AuthenticationAnswer
+    {
+        public UserRedis userRedis { get; set; }
+
+        public bool wrongPassword { get; set; } = false;
```

Comment:
For properties in `c#` we are using _PascalCase_. So we should Capitalize to `WrongPassword`